starter-code/src/Content.css

```diff
@@ -0,0 +1,21 @@
+
+.Content{
+ margin: 70px 100px 100px 100px;
```

Comment:
Watch out with the usage of "px" here. 

Due to your margin here this block is displayed with overflow on the page.